po/R-es.po

```diff
@@ -14,83 +14,83 @@ msgstr ""
 
 #: build-404.R:11
 msgid "Page not found (404)"
-msgstr "Página desconocida (404)"
+msgstr "Página no encontrada (404)"
 
 #: build-404.R:12
 msgid "Content not found. Please use links in the navbar."
-msgstr "Contenido desconocido. Por favor utilize los enlaces ubicados en la barra de navegación"
+msgstr "Contenido no encontrado. Por favor utilice los enlaces de la barra de navegación"
 
 #: build-articles.R:371 navbar.R:134 navbar.R:141 navbar.R:156
 msgid "Articles"
-msgstr "Guías"
+msgstr "Artículos"
 
 #: build-footer.R:29
 msgid "Developed by"
-msgstr "Creado por"
+msgstr "Desarrollado por"
 
 #: build-footer.R:34
 msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
-msgstr "Este sitio fue construido utilizando <a href=\"%s\">pkgdown</a> %s."
+msgstr "Sitio creado con <a href=\"%s\">pkgdown</a> %s."
 
 #: build-home-authors.R:92
 msgid "More about authors..."
 msgstr "Más sobre los autores..."
 
 #: build-home-authors.R:95
 msgid "Developers"
-msgstr "Autores"
+msgstr "Desarrolladores"
 
 #: build-home-authors.R:108 render.R:126
 msgid "Authors"
 msgstr "Autores"
 
 #: build-home-authors.R:200
 msgid "author"
-msgstr "Autor"
+msgstr "autor(a)"
 
 #: build-home-authors.R:201
 msgid "compiler"
-msgstr "compilador"
+msgstr "compilador(a)"
 
 #: build-home-authors.R:202
 msgid "contractor"
-msgstr "contratador"
+msgstr "contratista"
 
 #: build-home-authors.R:203
 msgid "contributor"
-msgstr "colaborador"
+msgstr "colaborador(a)"
 
 #: build-home-authors.R:204
 msgid "copyright holder"
-msgstr "dueño de los derechos de autor"
+msgstr "propietario de los derechos de autor"
```

Comment:
quizás acá habría que hacer la duplicación de género gramatical también, para mantener la consistencia con los otros roles

Reply:
I replaced this with "titular de los derechos de autor"

En España este es el término legal, pero me parecía apropiado ya que estamos hablando de derechos de autor. ¿Que te parece @rivaquiroga ?

Reply:
¡Mucho mejor!